string arm
geometry_msgs/Pose end_effector_frame
---
float64[] joint_positions
